### PR TITLE
Added API to retrieve user information

### PR DESCRIPTION
**Pull request by bexsoft**
## What does this do?
Added an API to retrieve user information from minioadmin

## How to test
1. Go into Postman, set verb as `GET` & URL as `api/v1/users/{user_name}`
2. Set Auth Key and send request
3. You should get the user information

### CODE REVIEW

File: restapi/admin_users.go [hunk 1]
```diff
@@ -65,6 +65,24 @@ func registerUsersHandlers(api *operations.McsAPI) {
...
 		return admin_api.NewUpdateUserGroupsOK().WithPayload(userUpdateResponse)
 	})
+	// Get User
+	api.AdminAPIGetUserInfoHandler = admin_api.GetUserInfoHandlerFunc(func(params admin_api.GetUserInfoParams, principal *models.Principal) middleware.Responder {
+		userInfoResponse, err := getUserInfoResponse(params)
+		if err != nil {
+			return admin_api.NewGetUserDefault(500).WithPayload(&models.Error{Code: 500, Message: swag.String(err.Error())})
+		}
+
+		log.Println(userInfoResponse)
```
Comment (cesnietor) [inline restapi/admin_users.go:75]:
```suggestion
```

File: restapi/admin_users.go [hunk 2]
```diff
@@ -65,6 +65,24 @@ func registerUsersHandlers(api *operations.McsAPI) {
...
+
+		userInformation := models.User{
+			AccessKey: params.Name,
+			MemberOf:  userInfoResponse.MemberOf,
+			Policy:    userInfoResponse.PolicyName,
+			Status:    string(userInfoResponse.Status),
+		}
```
Comment (cesnietor) [inline restapi/admin_users.go:82]: can you please serialize the output on the getUserInfoResponse? usually we use `..Response` functions to serialize the output. We try not to put more code than needed on the handler itself.

Comment (bexsoft) [reply restapi/admin_users.go]: Removed code from handler & moved into getUserInfoResponse function

**Comment by bexsoft** [comment]
Closing this PR to be merged in #51

**Pull request closed without merging**